frontend/src/components/indexPage/FilterSection.js

```diff
@@ -49,29 +50,8 @@ const searchBarLabels = {
   members: "Search for people active against climate change",
 };
 
-const buildUrlEndingFromSearch = (searchValue) => {
-  return "&search=" + searchValue;
-};
-
-export default function FilterSection({
-  filtersExpanded,
-  setFiltersExpanded,
-  typesByTabValue,
-  tabValue,
-  getProjects,
-  getOrganizations,
-  getMembers,
-  membersWithAdditionalInfo,
-  token,
```

Comment:
Many of these props weren't actually being used and just plucked out of the object. Removed them entirely and removed them from their call site within `browse`.